Vegeting/Screens/GroupList/ViewController/GroupListViewController.swift

```diff
@@ -0,0 +1,99 @@
+                    Club(clubID: "", clubTitle: "남미플랜트랩하고 거북이 먹으러 가요~!", clubCategory: "a", hostID: "a", participants: participants, createdAt: Date(), maxNumberOfPeople: 13),
+                    Club(clubID: "", clubTitle: "같이 비건 식당 탐방하실 분?", clubCategory: "a", hostID: "a", participants: participants, createdAt: Date(), maxNumberOfPeople: 14),
+                    Club(clubID: "", clubTitle: "동물권 관련 스터디", clubCategory: "a", hostID: "a", participants: participants, createdAt: Date(), maxNumberOfPeople: 15),
+                    Club(clubID: "", clubTitle: "기사 읽어보실 분~~", clubCategory: "a", hostID: "a", participants: participants, createdAt: Date(), maxNumberOfPeople: 16),
+                    Club(clubID: "", clubTitle: "비건 카페 가실분 계신가요?", clubCategory: "a", hostID: "a", participants: participants, createdAt: Date(), maxNumberOfPeople: 17)
+        ]
+    }
+    
+    private func configureCollectionView() {
+        collectionView.delegate = self
+        collectionView.dataSource = self
+    }
+    
+    private func configureUI() {
+        view.backgroundColor = .backgroundColor
+    }
+    
+    private func setupLayout() {
+        [collectionView].forEach {
+            view.addSubview($0)
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+        
+        NSLayoutConstraint.activate([
+            collectionView.topAnchor.constraint(equalTo: view.topAnchor),
+            collectionView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            collectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            collectionView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20)
+        ])
+    }
+}
+
+extension GroupListViewController: UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        print(clubList[indexPath.item])
+    }
+}
+
+extension GroupListViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return clubList.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ClubListCollectionViewCell.className, for: indexPath) as? ClubListCollectionViewCell else { return UICollectionViewCell() }
+        cell.configure(with: clubList[indexPath.item])
+        cell.layer.backgroundColor = UIColor.init(hex: "#e3e3e3").cgColor
+        return cell
+    }
+}
+
+extension GroupListViewController: UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        return CGSize(width: (collectionView.frame.width - 20) / 2, height: 210)
+    }
```

Comment:
p2;

저희 코딩컨벤션에 따라 매개변수가 3개 이상인 경우 각 파라미터를 새로운 라인으로 만들어주면 더 깔끔한 코드가 될 것 같습니다!
```suggestion
    func collectionView(_ collectionView: UICollectionView,
     layout collectionViewLayout: UICollectionViewLayout, 
     sizeForItemAt indexPath: IndexPath) -> CGSize {
        return CGSize(width: (collectionView.frame.width - 20) / 2, height: 210)
    }
```

Reply:
[dc9bdfe](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/56/commits/dc9bdfef53dbf211f48a29d38415ba0baa1a1a52)
반영완료!